app/models/inherited_proofing/audit_data.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module InheritedProofing
```

Comment:
Can you create a spec file for this? It's kinda dumb to do it for such a small class but I like them being there to encourage specs if we ever add to this model